docs/pages/guides/using-sentry.md

```diff
@@ -55,6 +55,8 @@ cmd={['$ expo install expo-application expo-constants expo-device expo-updates @
 
 ### Step 2: Code
 
+#### Configuration
+
 Add the following to your app's main file (usually `App.js`):
```

Comment:
```suggestion
Add the following to your app's main file such as **App.js**:
```

Unrelated to your PR, here is a small change to [bold the filenames as per Expo's writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md).